src/series_sum.py

```diff
@@ -0,0 +1,9 @@
+"""A function to find the sum of the first nth terms of a series 1+1/4+1/7+..."""
+
+
+def series_sum(n):
+    """Find the sum of the first nth terms of a series 1+1/4+1/7+..."""
+    sum = 0
+    for i in range(n):
+        sum += float(1 / (1 + i * 3))
```

Comment:
In Python 2, this is integer division even if you use the `float` type object to turn it into a float after the fact. Thus, all your answers will be "1"